### ISSUE TITLE
Polish translation for printing pop-up in document editor

### ISSUE THREAD

**Issue by ziomek64** [ONLYOFFICE/DesktopEditors#1005]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package), Linux (RPM package)

### Version information

7.1.1

### Expected Behavior

Have it translated

### Actual Behavior

It is not translated. Where do I look for those lines? I mean stuff like Print all, pages from to, page set etc. I could translate it

**Comment by ShockwaveNN** [comment]
Hi, could you clarify a window you've talked about

This one?
![image](https://user-images.githubusercontent.com/668524/187020447-a05345c8-8c8b-4966-9f64-5b38c4ad38a1.png)

Because this is system window, not sure that we've got control other that

**Comment by ziomek64** [comment]
@ShockwaveNN  Yes, this one, as you can see only title is translated. This comes from KDE then?

**Comment by ShockwaveNN** [comment]
I think it's something from QT even

Do you have system lang set to Eng like on my system on which screenshot made? Maybe this is the reason

**Comment by ziomek64** [comment]
@ShockwaveNN No, I have whole system in Polish.

**Comment by ShockwaveNN** [comment]
Ok, thanks we'll look into that, probably some bug or we simply cannot change this window, I'm almost sure this is system window

**Comment by ziomek64** [comment]
It could be! I'm not denying that. But you simply don't know where to start looking so I started here.
